manifests/u/uutils/findutils/0.7.0/uutils.findutils.installer.yaml

```diff
@@ -0,0 +1,20 @@
+# Created with komac v2.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: uutils.findutils
+PackageVersion: 0.7.0
+InstallerType: zip
+NestedInstallerType: portable
+# Should exclude testing-commandline.exe. See https://github.com/uutils/findutils/issues/19
+NestedInstallerFiles:
+- RelativeFilePath: find.exe
+- RelativeFilePath: xargs.exe

```

Comment:
Cannot exclude testing-commandline.exe with this definition because of they are in and installed into same directory.

Can't we exclude these files in winget phase?

Reply:
- https://github.com/microsoft/winget-pkgs/blob/2e025beaad105952eec7e2b3c1311950479958a3/doc/manifest/schema/1.9.0/installer.md
- https://github.com/microsoft/winget-cli/blob/6ec90f29b13373ebbb944d6d9b49dd134139afbf/doc/specs/%23182%20-%20Support%20for%20installation%20of%20portable%20standalone%20apps.md
- https://github.com/microsoft/winget-cli/blob/6ec90f29b13373ebbb944d6d9b49dd134139afbf/doc/specs/%23140%20-%20ZIP%20Support.md

I might have overlooked, but I couldn't find the solution in these documents.

Reply:
This might be an known problem? https://github.com/microsoft/winget-cli/issues/2909#issuecomment-1417967802